.gitignore

```
@@ -1,5 +1,7 @@
 __pycache__
 
+eeg_notebooks.egg-info/
```

This could be *.egg-info, but just nitpicking :)

```suggestion
*.egg-info/
```

---

Looked up about Python "eggs" and made this change!